recipes/z5py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: z5py
+  version: "0.5.2"
+
+source:
+    git_url: https://github.com/constantinpape/z5.git
+    git_rev: 0.5.2
+
+build:
+  number: 0
+  skip: True  # [win and vc<14 or not py36]
+
+requirements:
+  build:
+    - python {{PY_VER}}*
```

Comment:
conda-forge has some mechanisms to [build for a matrix of different versions]( https://conda-forge.org/docs/meta.html#build-matrices ) when appropriate. With `python`, that maps to [this list]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/d1dd6b5aeb3349c91079b6078d844760a788f302/recipe/conda_build_config.yaml#L405-L408 ). So `PY_VER` need not be specified and this can just be `python`. Our tooling will make sure these are correctly pinned for you. This applies to the usage below as well.